client/my-sites/site-settings/form-general.jsx

```diff
@@ -170,6 +171,23 @@ export class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
+	toolbarOption() {
+		const { isRequestingSettings, isSavingSettings, siteIsJetpack, siteIsAtomic } = this.props;
```

Comment:
Noting that while the `siteIsAtomic` prop doesn't seem to be provided here, it is provided by the `wrapSettingsForm` high-order component, so using it here is just fine 👍